dask_kubernetes/core.py

```diff
@@ -508,6 +508,35 @@ def f(to_close):
     def scale_up(self, n, **kwargs):
         self.scheduler.loop.add_callback(self._scale_up, n, **kwargs)
 
+    async def _pod_status(self, pods):
+
+        status = {}
+        for p in pods:
+            status[p.metadata.name] = p.status.phase
+        return status
+
+    async def check_status(self):
+        while True:
+            pods = await self._pods()
+            p_status = await self._pod_status(pods)
+            result = ['Running' == status for name, status in p_status.items()]
+            await asyncio.sleep(3.2)
```

Comment:
This seems like an odd place for a sleep, and also an odd value.  

I would expect to sleep only if we didn't pass everything, and I would also hope that the iteration time would be smaller than 3s.